app/controllers/users.js

```diff
@@ -0,0 +1,20 @@
+const logger = require('../logger');
+const userServices = require('../services/users');
+const { notFoundError } = require('../errors');
+
+exports.signUp = async (req, res, next) => {
+  try {
+    const { body } = req;
+    const userCreated = await userServices.signUpUser(body);
+
+    if (!userCreated && !userCreated.id) {
+      notFoundError('User not registered');
```

Comment:
use `return false` is very well buy y think that it's better to use `return next(notFoundError('User not registered'));`. What do you think ?